spec/system/jobseekers_can_subscribe_to_an_nqt_job_alert_spec.rb

```diff
@@ -6,20 +6,21 @@
   end
 
   describe "A jobseeker" do
```

Comment:
I think this whole system spec file should be deleted now

Reply:
No need to test this journey at all?

Reply:
As long as there is a request spec we're all good I think.

This journey is the same as creating a job alert from a search - which already exists